Library/ViewModels/PledgeViewModel.swift

```diff
@@ -141,7 +147,12 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     self.pledgeAmountObserver.send(value: amount)
   }
 
-  private let (shippingRuleSignal, shippingRuleObserver) = Signal<ShippingRule, Never>.pipe()
+  private let presentShippingRulesProperty = MutableProperty<(ShippingRule)?>(nil)
+  public func presentShippingRules(with rule: ShippingRule) {
```

Comment:
Added new signal to present shipping rules...the old signal was more appropriate for when we actually select a rule..I've kept it around because there's bunch of test related to that...this should be a plumbing for the selection itself.